src/constants/mapIcons.js

```diff
@@ -1,55 +1,19 @@
 export default {
-  compile: [
-    'cc',
-    'ccache',
-    'clang',
-    'gcc',
-    'gmake',
-    'make',
-    'xcodebuild',
-  ],
-  clojure: [
-    'lein',
-    'planck',
-    'lumo',
-  ],
-  docker: [
-    'docker-compose',
-  ],
+  compile: ['cc', 'ccache', 'clang', 'gcc', 'gmake', 'make', 'xcodebuild'],
+  clojure: ['lein', 'planck', 'lumo'],
+  docker: ['docker-compose'],
   git: [
     'git-remote-ftp',
     'git-remote-ftps',
     'git-remote-http',
-    'git-remote-https',
-  ],
-  http: [
-    'wget',
-    'http',
-  ],
-  monitor: [
-    'htop',
-    'iftop',
-    'top',
-  ],
-  nodejs: [
-    'node',
-  ],
-  ruby: [
-    'irb',
-    'sidekiq',
-    'rake',
-  ],
-  shell: [
-    'bash',
-    'fish',
-    'zsh',
-  ],
-  code: [
-    'pico',
-  ],
-  read: [
-    'tail',
-    'less',
-    'more',
-  ],
+    'git-remote-https'
+  ],
+  http: ['wget', 'http'],
+  monitor: ['htop', 'iftop', 'top'],
+  database: ['mongodb', 'mongod', 'mongo', 'mysql'],
+  nodejs: ['node'],
+  ruby: ['irb', 'sidekiq', 'rake'],
+  shell: ['bash', 'fish', 'zsh'],
+  code: ['pico'],
+  read: ['tail', 'less', 'more']
 };
```

Comment:
Please lint this file, you can run `npm run lint` to find errors.

Reply:
Done this.